html/about.html

```diff
@@ -131,6 +133,21 @@ <h3 class="about-name-surname">Krystian Borkowski</h3>
                             ></a>
                         </div>
                     </div>
+                    <div class="col-12 col-md-4">
```

Comment:
i'd think about rearranging the columns as col-12 is the maximum and with 4x col-md-4 we've got sth like col-16, which won't work well probably, so we can divide everything into 2 big columns col-md-6 (probably), or into 4 smaller ones (col-md-3)

Reply:
good idea, I"ll change it right away!